README.md

```diff
@@ -89,24 +89,25 @@ npx npm-merge-driver install --global
 ----
 
 # To start the server:
-- ssh onto google cloud 
-- go to the folder containing the repository
-- run
-export PORT=3000
+- ssh onto the google cloud server
+- Go to the folder containing the repository
+- Run `export PORT=3000`. In your linux terminal, this sets up the port for the project to use (as an environment variable).
+
+## Temporaty hosting
 ```
-In your linux terminal, this sets up the port for the project to use.
-- for a temporary hosting use 
 npm run serve
 ```
-- for a persistent hosting use
-nohup npm run serve
+
+## Persistent hosting
+- Run `npm run build` to build the frontend into a set of static files (the result is in the `dist` folder)
+- Start the server:
 ```
-This tells ubuntu not to kill the process when we log out.
-- Then to kill the process: run 
-ps ux
+nohup npm run production
 ```
-Look for the 2 processes labelled 'node home/.../vue-cli-...' and 'node home/.../nodemon server/index.js', get their IDs.
-- Run '$Kill -9 ID' for both of them
+- `nohup` tells ubuntu not to kill the process when we log out
+- Then to kill the process run `ps ux`
+- Look for the 2 processes labelled `node home/.../vue-cli-...` and `node home/.../nodemon server/index.js`, get their IDs
+- Run `$Kill -9 ID` for both of them

```

Comment:
Outdated info, we should talk about using tmux to host as seen here: https://askubuntu.com/questions/8653/how-to-keep-processes-running-after-ending-ssh-session